### DISCUSSION TITLE
5.35 file doesn't have exe  in it.

### DISCUSSION THREAD

**Discussion by ww3459** [kangyu-california/PersistentWindows#162] [General]
tried to upgrade from 5.34 to 5.35 but cmd file didnt upgrade it. so i downloaded 5.35 version deleted the 5.34 but 5.35 file didnt have any executable exe file in it to make it program to boot with windows. help pls

**Comment by kangyu-california** [comment]
The 5.35 release is fine, there are 3 links (2 of them are source code and automatically composed by github), only the first one contains binary https://github.com/kangyu-california/PersistentWindows/releases/download/5.35/PersistentWindows5.35.zip
You may also upgrade from 5.34 by menu "upgrade to 5.35", which should succeed if you run 5.34 with admin privilege
